### PR TITLE
Convert William Kielar to SAI

### PR DESCRIPTION
**Pull request by mysterious-static**
As discussed in #16112 and modelled off of @tkrokli's work in #15966, this moves the NPC William Kielar to SAI. I tested it and it seems to still be fully functional.

Eager for feedback; if I'm doing this more or less correctly, I'll start getting some more NPC's migrated per @Aokromes list from https://github.com/TrinityCore/TrinityCore/pull/15966#issuecomment-167864140

### CODE REVIEW

File: sql/updates/world/2015_12_31_XX_world.sql [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+DELETE FROM `smart_scripts` WHERE (source_type = 0 AND entryorguid = 17209);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(17209, 0, 0, 3, 62, 0, 100, 0, 7379, 0, 0, 0, 52, 494, 0, 0, 0, 0, 0, 7, 0, 0, 0, 0, 0, 0, 0, 'William Kielar - On Gossip Option 0 Selected - Activate Taxi Path 494'),
+(17209, 0, 1, 3, 62, 0, 100, 0, 7379, 1, 0, 0, 52, 495, 0, 0, 0, 0, 0, 7, 0, 0, 0, 0, 0, 0, 0, 'William Kielar - On Gossip Option 1 Selected - Activate Taxi Path 495'),
+(17209, 0, 2, 3, 62, 0, 100, 0, 7379, 2, 0, 0, 52, 496, 0, 0, 0, 0, 0, 7, 0, 0, 0, 0, 0, 0, 0, 'William Kielar - On Gossip Option 2 Selected - Activate Taxi Path 496'),
+(17209, 0, 3, 0, 61, 0, 100, 0, 0, 0, 0, 0, 72, 0, 0, 0, 0, 0, 0, 7, 0, 0, 0, 0, 0, 0, 0, 'William Kielar - On Gossip Option 0 Selected - Close Gossip');
```
Comment (Vincent-Michael) [inline sql/updates/world/2015_12_31_XX_world.sql]: fix pls end of line

Comment (velinath) [inline sql/updates/world/2015_12_31_XX_world.sql]: Just newline at end or that L14 is obscenely long?

Comment (DDuarte) [inline sql/updates/world/2015_12_31_XX_world.sql]: Yes, it's just an empty line at the end of the file

Comment (velinath) [inline sql/updates/world/2015_12_31_XX_world.sql]: All set!

**Comment by Killyana** [comment]
Looks good for me.

**Comment by velinath** [comment]
Trying to squash, but it's throwing an error. Might take a bit to resolve so if you see merge conflicts pop up that's probably why

**Comment by velinath** [comment]
Super sorry about that, rebase should be all set

**Comment by tkrokli** [comment]
Nice job, @velinath ! Looking forward to see more of this. :+1:

**Comment by Treeston** [comment]
Yeah, we can always use more DB devs. We have too few of those as is.

**Pull request merged**